### ISSUE TITLE
NVDA does not read comments/notes/annotations created with Adobe Reader

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#12525]
It is not possible to have the notes or annotations in a PDF document read aloud if these notes/annotations have been created with Adobe Reader's own tools.

Unfortunately, the comment list in Adobe Reader is not accessible, so there is no other way to have the comments read aloud.

NVDA 2020.4

Windows
Edition Windows 10 Home
Version	20H2
Installed on ‎05.‎02.‎2021
Operating system build	19042.985

LibreOffice
Version: 7.1.3.2 (x64) / LibreOffice Community
Build ID: 47f78053abe362b9384784d31a6e56f8511eb1c1
CPU threads: 4; OS: Windows 10.0 Build 19042; UI render: Skia/Raster; VCL: win
Locale: de-DE (de_DE); UI: de-DE
Calc: threaded

**Comment by Qchristensen** [comment]
This is also the case for PDF files with comments created in Microsoft Word.  Sample PDF attached:
[test.pdf](https://github.com/nvaccess/nvda/files/11196856/test.pdf)


NVDA 2023.1
 
Windows 11 (64-bit) Version: 22H2, Build: 22621.1413 
 
Office 365 (64-bit) Version: 16.0.16227.20244
